cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -748,8 +758,18 @@ private V1ObjectMeta createStatefulSet(V1ObjectMeta metadata,
 
     V1StatefulSet statefulSet = buildStatefulSet(metadata, runtimeSpecs, runtimeConfigLocation, statefulRunnable);
 
-    statefulSet = appsApi.createNamespacedStatefulSet(programRuntimeNamespace, statefulSet, "true", null, null);
-    LOG.info("Created StatefulSet {} in Kubernetes", metadata.getName());
+    try {
+      statefulSet = appsApi.createNamespacedStatefulSet(programRuntimeNamespace, statefulSet, "true", null, null);
+      LOG.info("Created StatefulSet {} in Kubernetes", metadata.getName());
+    } catch (ApiException e) {
+      if (e.getCode() == HttpURLConnection.HTTP_CONFLICT) {
+        LOG.warn("The kubernetes statefulset already exists : {}. Ignoring resubmission of the statefulset.",
+                 e.getResponseBody());
+      } else {
+        LOG.error("Failed to create kubernetes statefulset: {}", e.getResponseBody());
```

Comment:
Skip the logging since the exception is rethrown.

Reply:
Fixed